tools/cockpit.spec

```diff
@@ -327,7 +327,7 @@ The Cockpit components for managing software updates for ostree based systems.
 
 %package pcp
 Summary: Cockpit PCP integration
-Requires: %{name}-bridge = %{version}-%{release}
+Requires: %{name}-bridge > 134
```

Comment:
Would it make sense to add a `"requires": {"cockpit": "134"}}` to pkg/pcp/manifest.json.in and use `%required_base` for the dependency instead?